modules/ztp-sno-du-enabling-workload-partitioning.adoc

```diff
@@ -40,7 +39,7 @@ If Hyper-Threading is enabled, specify both threads for each core. The `CPUs` va
   }
 }
 ----
-<1> The `cpuset` must match the `CPUs` value in `/etc/crio/crio.conf.d/01-workload-partitioning`.
+<1> The `cpuset` must match the `cpuset` value in `/etc/crio/crio.conf.d/01-workload-partitioning`.
```

Comment:
Hey folks, just to clarify the CRI-O config is read in, but the current kubelet implementation for the file `/etc/kubernetes/openshift-workload-pinning` doesn't read the contents just yet, but that might change in the future and it's expected that, that file will contain that information so I would suggest we continue to have it in the docs to avoid any confusion.

Reply:
Ack. thanks. So this is OK as is.